### DISCUSSION TITLE
Unable to delete a project

### DISCUSSION THREAD

**Discussion by CodeMeister** [testspace-com/support#24] [Q&A] [answered]
I have a project for a repo that has now been deleted.

When selecting the project, the hamburger menu has the 'delete' option muted and displays the 'not-allowed' cursor.

Given I am the project/account owner, what else is needed?

**Comment by munderseth** [comment, marked as answer]
Hi @CodeMeister can you double-check that you are using the `proeject` hamburger menu vs the `branch`?

![image](https://user-images.githubusercontent.com/1229367/131569535-61130bdb-56d9-4ef5-a3dd-607d265304bd.png)

The `branch` can not be deleted, it tracks the repo.

![image](https://user-images.githubusercontent.com/1229367/131569686-7e1a9716-4be3-484f-9938-287d392f2f24.png)

**Reply by CodeMeister** [reply]
Thanks @munderseth, my bad :)
